Milyli.ScriptRunner.Services.Contracts/V1/ScriptRunHistory.cs

```diff
@@ -1,11 +1,11 @@
-﻿namespace Milyli.ScriptRunner.Services.Interfaces.V1.Models.Responses
+﻿namespace Milyli.ScriptRunner.Services.Contracts.V1
 {
 	using System;
 
 	/// <summary>
-	/// Response to Script Run history requests
+	/// Defines a single past execution of a Script Run.
 	/// </summary>
-	public class ReadRunHistoryResponse
+	public class ScriptRunHistory
 	{
 		/// <summary>
 		/// Script Run History identifier.
```

Comment:
Is the user every going to need the Id of the ScriptRunHistory object?

  - [ ] Possibly remove the Id from the model

Reply:
Somewhat related, if the Input doesn't have the ScriptRunId than this model probably shouldn't either.

  - [ ] Make the ScriptRunId consistent between the models